### ISSUE TITLE
Error using -r flag

### ISSUE THREAD

**Issue by Ge0rges** [rhysnewell/aviary#164]
When using the -r flag to filter against a reference I get the following errors:

```
[E::hts_open_format] Failed to open file "data/raw_mapped_ref.bai" : No such file or directory
samtools view: failed to open "data/raw_mapped_ref.bai" for reading: No such file or directory
```

That is preceded by 

```
rule map_reads_ref:
input: /researchdrive/gkanaan/seaice_methylation/fastas/raw/barcode01.fasta, /researchdrive/gkanaan/seaice_methylation/fastqs/raw/barcode02.fasta, /resear chdrive/gkanaan/seaice_methylation/fastqs/raw/barcode03.fasta, /researchdrive/gkanaan/seaice_methylation/fastqs/raw/barcode05.fasta, /researchdrive/gkanaan/se
aice_methylation/fastas/raw/barcode06.fasta, /researchdrive/gkanaan/seaice_methylation/fastqs/raw/barcode07.fasta, /researchdrive/gkanaan/seaice_methylation/f
astas/raw/barcode08.fasta, /researchdrive/gkanaan/seaice_methylation/fastqs/raw/barcode09.fasta, /researchdrive/gkanaan/seaice_methylation/fastqs/raw/barcode1 O.fasta, /researchdrive/gkanaan/seaice_methylation/fastqs/raw/barcode11.fasta, /researchdrive/gkanaan/seaice_methylation/fastqs/raw/barcode12.fasta, /research drive/gkanaan/seaice_methylation/fastqs/raw/barcode13.fasta, /researchdrive/gkanaan/seaice_methylation/fastqs/raw/barcode14.fasta, /researchdrive/gkanaan/data bases/human_fasta/GCF.
_000001405.40_GRCh38.p14_genomic. fna
output: data/raw_mapped_
_ref.bam, data/raw_mapped_ _ref.bal
jobid: 5
benchmark: benchmarks/map_reads_ref.benchmark.txt
threads: 30
Activating conda environment: /us/local/miniconda3/envs/8f0021a1ebf3e319564f21ffc3eb0a78
```

**Comment by rhysnewell** [comment]
Hey @Ge0rges, 

Could you post the log files in the log directory and which version you are using?

**Comment by rhysnewell** [comment]
Any chance to grab that version information? Otherwise I'll have to close this issue

**Comment by Ge0rges** [comment]
I believe it was v0.8.1

**Comment by rhysnewell** [comment]
I think this would be fixed with v0.8.2
